setup.py

```diff
@@ -1,6 +1,6 @@
 from distutils.core import setup
 
-setup(name='GeneralPretrainedModelTools',
+setup(name='Apollo',
       version='1.0',
-      packages=["cdm_processing", "utils"],
+      packages=["cdm_processing", "utils", "simulation", "data_generation"],
```

Comment:
There's a mismatch here. The folder with data_generation code is called "data_generating". This gives an error when trying to install the package. I would be in favor of using "data_generation", so renaming the folder.

Reply:
Agreed. For consistency, I'll rename `simulation` to `simulating`, and `simulation.py` to `simulator.py` as well.